contracts/src/systems/hyperstructure/transport_to_hyperstructure.cairo

```diff
@@ -0,0 +1,75 @@
+#[system]
+mod TransportToHyperStructure {
```

Comment:
I think this system could be generalised with the transport system, as it is only shipping from a->b

Reply:
so the flow for hyperstructures using 1-way Travels would be:

1. Travel Realm -> Hyperstruct (then wait for the caravan to arrive)
2. Offload Caravan
3. Travel back to Realm 

Using 2-way Travels you could have:
1. Travel back and forth from Realm -> Hyperstructure -> Realm.
2. Offload Caravan when we know the caravan is on the way back from the Hyperstructure

Option 2 is similar to what is done in trading (albeit not generalizable to other cases yet)

Would be interesting to discuss both approaches together. @credence0x @ponderingdemocritus